platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -712,7 +712,7 @@ public void onCreate(Bundle icicle) {
 				notifierIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP);
 
 				PendingIntent pendingIntent = PendingIntent.getActivity(activity, 0,
-						notifierIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+						notifierIntent, PendingIntent.FLAG_UPDATE_CURRENT | PendingIntent.FLAG_IMMUTABLE);
```

Comment:
The added flag was introduced in api 23 and our min is 21, so you need to add a conditional check prior to using it 

https://developer.android.com/reference/android/app/PendingIntent#FLAG_IMMUTABLE